internal/database/online/dynamodb/stream.go

```diff
@@ -39,3 +40,41 @@ func (db *DB) PrepareStreamTable(ctx context.Context, opt online.PrepareStreamTa
 	})
 	return err
 }
+
+func (db *DB) Push(ctx context.Context, opt online.PushOpt) error {
+	var (
+		tableName = sqlutil.OnlineStreamTableName(opt.GroupID)
+		item      = make(map[string]types.AttributeValue)
+	)
+
+	entityKeyValue, err := attributevalue.Marshal(opt.EntityKey)
+	if err != nil {
+		return err
+	}
+	item[opt.Entity.Name] = entityKeyValue
+
+	for i, feature := range opt.Features {
+		value, err := serializeByTag(opt.FeatureValues[i], feature.ValueType)
+		if err != nil {
+			return err
+		}
+		attributevalue, err := attributevalue.Marshal(value)
+		if err != nil {
+			return err
+		}
+		item[feature.Name] = attributevalue
+	}
+
+	_, err = db.BatchWriteItem(ctx, &dynamodb.BatchWriteItemInput{
```

Comment:
Use `PutItem` instead? Since it is not a batch write

Reply:
done